gap/examples.gi

```diff
@@ -370,3 +370,51 @@ GeneralisedPetersenGraphCons);
 
 InstallMethod(GeneralisedPetersenGraph, "for integer, integer", [IsInt, IsInt],
 {n, k} -> GeneralisedPetersenGraphCons(IsImmutableDigraph, n, k));
+
+InstallMethod(KnightsGraphCons,
+"for IsMutableDigraph and two positive integers",
+[IsMutableDigraph, IsPosInt, IsPosInt],
+function(filt, n, k)
+  local D, moveOffSets, coordinates, targetCoordinate, p, i, j;
+  D := EmptyDigraph(IsMutableDigraph, n * k);
+  moveOffSets := [[2, 1], [-2, 1], [2, -1], [-2, -1],
+  [1, 2], [-1, 2], [1, -2], [-1, -2]];
+  coordinates := [];
+  p := 0;
+
+  for i in [1 .. k] do
+    for j in [1 .. n] do
+      p := p + 1;
+      coordinates[p] := [i, j];
+    od;
+  od;
```

Comment:
So basically `QuotientRemainder(p, n) = [i, j]` if and only if `coordinates[p] = [i + 1, j]`? (except remainder would be 0 rather than 5)